index.html

```diff
@@ -17,58 +17,42 @@
         <div class="card">
             <h5 class="card-header">Educational Details</h5>
             <div class="card-body">
-                <div class="detail">
-                    <div class="row">
-                        <div class="col-6">
-                            <h6>Master of Technology in Computer Science </h6>
-                        </div>
-                        <div class="col-6">
-                           
-                            <label class="fw-bolder">  May 2024</label>
-
-                        </div>
-                        <div class="col-12 mb-2">
-                           <strong class="text-capitalize"> pandit shivdatta shastri education foundation's asma institute of management Shivane Pune</strong>
-                        </div>
-                               <hr>
-                        <div class="col-6">
-                            <h6>Bachelore of Technology in Computer Science </h6>
-                        </div>
-                        <div class="col-6">
-                            
-                            <label class="fw-bolder"> May 2020</label>
-
-                        </div>
-                        <div class="col-12 mb-2">
-                           <strong>Krantishih Nana Patil College Walwa Sangli</strong>
-                        </div>
-                        <hr>
-                       
-                        <div class="col-6">
-                            <h6>Higher Secondary Education (12<sup>th</sup> Grade) </h6>
-                        </div>
-                        <div class="col-6">
-                           
-                            <label class="fw-bolder"> March 2017</label>
-                        </div>
-                        <div class="col-12 mb-2">
-                            <strong class="text-capitalize"> padmabhushan dr. vasantraodada patil mahavidyalaya tasgaon Sangli</strong>
-                         </div>
-                         <hr>
-                        <div class="col-6">
-                            <h6> Secondary Education (10<sup>th</sup> Grade) </h6>
-                        </div>
-                        <div class="col-6">
-                            <label class="fw-bolder">Feb 2015</label>
-                        </div>
-                        <div class="col-12 mb-2">
-                            <strong class="text-capitalize">  Panchkroshi Vidyaniketan Nimani tasgaon Sangli</strong>
-                         </div>
-                         <hr>
-                    </div>
-                   
-
-                </div>
+                <table class="table table-dark table-striped table-hover">
+                    <thead>
+                      <tr>
+                        <th scope="col">Passout Year</th>
+                        <th scope="col">Examination</th>
+                        <th scope="col">Institute</th>
+                        <th scope="col">Sgpa/ Percentage</th>
```

Comment:
Keep only percentage . If all marks in percentage format

Reply:
sgpa are resolved